packages/frontend-design-poc/src/components/FilterBar/FilterBar.tsx

```diff
@@ -134,6 +139,20 @@ export const FilterBar = ({ onFilterChange, fields, initialFilters = [] }: Filte
     [activeFilters, onFilterChange],
   );
 
+  const handleSaveSearch = () => {
+    const searchHistory: SavedSearch[] = getSearchHistory();
+    const newSearch: SavedSearch = {
+      id: Date.now(),
+      filters: activeFilters,
+      timestamp: new Date(),
```

Comment:
datoer bør lagres som iso-strenger. så `.toISOString()` for da har man alltid en parsable representasjon som streng i UTC med offset.